.github/workflows/publish_agentkit.yml

```diff
@@ -0,0 +1,51 @@
+name: Release AgentKit to PyPI
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy-agentkit:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cdp-agentkit-core
+    environment:
+      name: pypi
+      url: https://pypi.org/p/cdp-agentkit-core
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+      with:
+        version: latest
+        virtualenvs-create: true
+        virtualenvs-in-project: true
+
+    - name: Load cached venv
+      id: cached-poetry-dependencies
+      uses: actions/cache@v3
+      with:
+        path: ./cdp-agentkit-core/.venv
+        key: venv-agentkit-${{ runner.os }}-3.10-${{ hashFiles('cdp-agentkit-core/poetry.lock') }}
+
+    - name: Install dependencies
+      if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
+      run: poetry install --with dev
```

Comment:
we don't want the dev dependencies in the build

Reply:
oh shoot, roger that!